src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs

```diff
@@ -327,25 +327,33 @@ internal static BaseProcessor<LogRecord> BuildOtlpLogExporter(
             experimentalOptions!);
 #pragma warning restore CA2000 // Dispose objects before losing scope
 
-        if (configureExporterInstance != null)
+        try
         {
-            otlpExporter = configureExporterInstance(otlpExporter);
-        }
+            if (configureExporterInstance != null)
+            {
+                otlpExporter = configureExporterInstance(otlpExporter);
+            }
 
-        if (processorOptions!.ExportProcessorType == ExportProcessorType.Simple)
-        {
-            return new SimpleLogRecordExportProcessor(otlpExporter);
+            if (processorOptions!.ExportProcessorType == ExportProcessorType.Simple)
+            {
+                return new SimpleLogRecordExportProcessor(otlpExporter);
+            }
+            else
+            {
+                var batchOptions = processorOptions.BatchExportProcessorOptions;
+
+                return new BatchLogRecordExportProcessor(
+                    otlpExporter,
+                    batchOptions.MaxQueueSize,
+                    batchOptions.ScheduledDelayMilliseconds,
+                    batchOptions.ExporterTimeoutMilliseconds,
+                    batchOptions.MaxExportBatchSize);
+            }
         }
-        else
+        catch
         {
-            var batchOptions = processorOptions.BatchExportProcessorOptions;
-
-            return new BatchLogRecordExportProcessor(
-                otlpExporter,
-                batchOptions.MaxQueueSize,
-                batchOptions.ScheduledDelayMilliseconds,
-                batchOptions.ExporterTimeoutMilliseconds,
-                batchOptions.MaxExportBatchSize);
+            otlpExporter.Dispose();
+            throw;
```

Comment:
How will the customer know what steps they need to take to resolve the issue based on this exception?